DistFiles/xMatter/bloom-xmatter-mixins.jade

```diff
@@ -97,10 +97,8 @@ mixin factoryStandard-outsideFrontCover
 		//- light of jade it is overly complicated.
 
 		+field-prototypeDeclaredExplicity.bookTitle
-			label.bubble Book title in {lang}
```

Comment:
This was prompted by the new bubble that is the subject of this PR, which now shows on top of this.  Ideally, we'd show this *unless* there was a source bubble. Hmmm.